commands/helpcenter.js

```diff
@@ -0,0 +1,50 @@
+const guildConfig = require('../util/guildConfig.js');
+const util = require('../lib/util.js');
+const axios = require('axios');
+
+exports.command = async (message, args, database, bot) => {
+    //Permission check
+    if (!message.member.hasPermission('MANAGE_GUILD')) {
+        await message.channel.send('You need the "Manage Server" permission to use this command.');
+        return;
+    }
+
+    if (!args[0]) {
+      await message.react(util.icons.error);
+      await message.channel.send("Please provide a Zendesk subdomain e.g. https://example.zendesk.com");
+      return;
+    }
+
+    let subdomain = args.shift().replace(/\.zendesk\.com$/i, '').replace(/[^a-zA-Z\d]/g, '');;
```

Comment:
Also replace http(s)://

```suggestion
    let subdomain = args.shift().replace(/^https?:\/\/|\.zendesk\.com(\/.*)?$/ig, '').replace(/[^a-zA-Z\d]/g, '');;
```